### PR TITLE
Support placement for popper.js

### LINKED ISSUE #64315: Support placement option for tour-kit
### What

- Support `placement` option for tour-kit. 

### Why

We need it for https://github.com/woocommerce/woocommerce/issues/32421

### How

_No response_

### PR DESCRIPTION
**Pull request by moon0326**
#### Proposed Changes

* Add `placement` option to control popper.js placement.

#### Testing Instructions

- Run `yarn workspace @automattic/tour-kit run storybook`
- Go to `tour-kit > Placement`
- Click "Start Tour" and confirm the effect work

Fixes #64315

### CODE REVIEW

File: packages/tour-kit/src/types.ts [hunk 1]
```diff
@@ -84,6 +84,19 @@ export interface Config {
 	closeHandler: CloseHandler;
 	isMinimized?: boolean;
 	options?: Options;
+	placement?:
```
Comment (chihsuan) [inline packages/tour-kit/src/types.ts]: @moon0326 I think we can use the `Placement` type from `@popperjs/core`. 

```ts
import * as PopperJS from '@popperjs/core';


	placement?: PopperJS.Placement;
```

Comment (chihsuan) [inline packages/tour-kit/src/types.ts]: @moon0326 I think we can use the `Placement` type from `@popperjs/core`. 

```ts
import * as PopperJS from '@popperjs/core';


	placement?: PopperJS.Placement;
```

**Review by chihsuan — COMMENTED** [review]
@moon0326 Thanks for adding this. Tested well!

I only left one comment about types and it would be great if you could update the documentation in the readme.md. Thanks!

**Comment by chihsuan** [comment]
cc @Automattic/vertex

**Comment by escapemanuele** [comment]
Well done, nice to have this!

File: packages/tour-kit/src/types.ts (hunk 1 above)
Comment (moon0326) [reply packages/tour-kit/src/types.ts]: Good suggestion! 👍 updated in https://github.com/Automattic/wp-calypso/pull/64317/commits/03cfe66350873cee27c4a4e775036a5f58268483

**Review by chihsuan — APPROVED** [review]
Pre-approving! @moon0326 Would you be able to update the documentation in the readme.md?

**Pull request merged**